bin/check_samplesheet.py

```diff
@@ -325,7 +325,7 @@ def main(argv=None):
     logging.basicConfig(level=args.log_level, format="[%(levelname)s] %(message)s")
     if not args.file_in.is_file():
         logger.error(f"The given input file {args.file_in} was not found!")
-        sys.exit(2)
+        sys.exit(f"The given input file {args.file_in} was not found!")
```

Comment:
Not sure if we should change this since this is coming from the template.

Reply:
you're right, it seems like some of those are fixed in the next template update